cypress/e2e/hub/repositories.cy.ts

```diff
@@ -8,4 +9,21 @@ describe('Repositories', () => {
     cy.navigateTo('hub', Repositories.url);
     cy.verifyPageTitle(Repositories.title);
   });
+  it('should copy CLI to clipboard', () => {
+    const repositoryName =
+      'repositories_repository_' + randomString(6, undefined, { isLowercase: true });
+    cy.navigateTo('hub', Repositories.url);
+    cy.verifyPageTitle(Repositories.title);
+    cy.get('[data-cy="create-repository"]').should('be.visible').click();
+    cy.get('[data-cy="name"]').type(repositoryName);
+    cy.get('[data-cy="Submit"]').click();
+    cy.get('[data-cy="actions-dropdown"]').click();
+    cy.get('[data-cy="copy-cli-configuration"]').click();
+    cy.get('[data-cy="alert-toaster"]').should('be.visible');
+    /* cy.window().then(win => {
```

Comment:
If we are creating a repository as part of this test, we should delete at the end.

Reply:
@nixocio Done :) Are you ok with merging it?